include/tinycompress/compress_ops.h

```diff
@@ -42,6 +42,7 @@ struct compress_ops {
 	int (*is_compress_running)(void *compress_data);
```

Comment:
You need to add your Signed-off-by tag. 

Also, maybe seems trivial but could you add some details about this patch. Why is it needed? Who uses it?